### ISSUE TITLE
Please bump version of fresh

### ISSUE THREAD

**Issue by vytautas-pranskunas-** [felangel/fresh#37]
Still have troubles to upgrade to new flutter

Because fresh_graphql >=0.3.0 depends on fresh ^0.3.0 and fresh_dio >=0.3.0-nullsafety.0 depends on fresh ^0.4.0, fresh_graphql >=0.3.0 is incompatible with fresh_dio >=0.3.0-nullsafety.0.

**Comment by vytautas-pranskunas-** [comment]
one more: 

Because every version of fresh_graphql depends on http ^0.12.1 which depends on http_parser >=0.0.1 <4.0.0, every version of fresh_graphql requires http_parser >=0.0.1 <4.0.0.
So, because senio_assist depends on both fresh_graphql ^0.4.0 and http_parser ^4.0.0, version solving fai

**Comment by vytautas-pranskunas-** [comment]
and one more:

Because image_picker >=0.7.0 depends on image_picker_platform_interface ^2.0.0 which depends on http ^0.13.0, image_picker >=0.7.0 requires http ^0.13.0.
And because every version of fresh_graphql depends on http ^0.12.1, image_picker >=0.7.0 is incompatible with fresh_graphql.

**Comment by vytautas-pranskunas-** [comment]
and one more:

Because hive_flutter >=1.0.0 depends on path_provider ^2.0.0 and graphql_flutter >=4.0.0-alpha.0 <5.0.0-nullsafety.1 depends on path_provider ^1.6.10, hive_flutter >=1.0.0 is incompatible with graphql_flutter   
  >=4.0.0-alpha.0 <5.0.0-nullsafety.1.
So, because senio_assist depends on both graphql_flutter ^4.0.0 and hive_flutter ^1.0.0, version solving failed.

**Comment by vytautas-pranskunas-** [comment]
One more:

Because graphql_flutter >=5.0.0-nullsafety.1 depends on graphql 5.0.0-nullsafety.1 and fresh_graphql >=0.4.0 depends on graphql ^4.0.0, graphql_flutter >=5.0.0-nullsafety.1 is incompatible with fresh_graphql    
  >=0.4.0.
So, because senio_assist depends on both fresh_graphql ^0.4.0 and graphql_flutter ^5.0.0-nullsafety.1, version solving failed.

**Comment by reilem** [comment]
Exact same issues here, cannot use new graphql because of this package

**Comment by tuanvugoodmoney** [comment]
@vytautas-pranskunas- @reilem See PR #41 and version 0.5.0 is published.

**Comment by vytautas-pranskunas-** [comment]
Thanks for your effort. Unfortunetly I had tried to upgrade my project without success and next attempt will be taken only after few month :( If @reilem you can try it would be super cool.

**Comment by reilem** [comment]
I've been able to upgrade but I now have a new issue: https://github.com/felangel/fresh/issues/42

**Comment by felangel** [comment]
Closing this since the versioning issue has been resolved and #42 should be fixed in [v0.5.1](https://pub.dev/packages/fresh_graphql) 👍
